src/main/java/com/iota/iri/network/replicator/ReplicatorSinkProcessor.java

```diff
@@ -93,12 +93,12 @@ public void run() {
                                             try {
                                                 CRC32 crc32 = new CRC32();                                        
                                                 crc32.update(message.array());
-                                                String crc32_string = Long.toHexString(crc32.getValue());
+                                                StringBuilder crc32_string = new StringBuilder(Long.toHexString(crc32.getValue()));
```

Comment:
I suggest just using StringUtils.leftPad(string,size,pad). Get rid of the concatenation altogether. StringUtils is already part of the required dependency chain.

Reply:
I agree, it will be cleaner.